src/otx/core/model/entity/classification.py

```diff
@@ -227,43 +226,25 @@ def _customize_outputs(
         )
 
 
-class OVClassificationCompatibleModel(OTXMulticlassClsModel):
+class OVMulticlassClassificationModel(OVModel):
     """Classification model compatible for OpenVINO IR inference.
 
     It can consume OpenVINO IR model path or model name from Intel OMZ repository
     and create the OTX classification model compatible for OTX testing pipeline.
     """
 
-    def __init__(self, config: DictConfig) -> None:
-        self.model_name = config.pop("model_name")
-        self.config = config
-        super().__init__()
-
-    def _create_model(self) -> nn.Module:
-        from openvino.model_api.models import ClassificationModel
-
-        return ClassificationModel.create_model(self.model_name, model_type="Classification")
-
-    def _customize_inputs(self, entity: MulticlassClsBatchDataEntity) -> dict[str, Any]:
-        if entity.batch_size > 1:
-            msg = "Only sync inference with batch = 1 is supported for now"
-            raise RuntimeError(msg)
-        # restore original numpy image
-        img = np.transpose(entity.images[-1].numpy(), (1, 2, 0))
-        return {"inputs": img}
-
     def _customize_outputs(
         self,
         outputs: Any,  # noqa: ANN401
-        inputs: MulticlassClsBatchDataEntity,
+        inputs: MultilabelClsBatchDataEntity,
```

Comment:
It seems wrong correction since it's classname is `OVMulticlassClassificationModel`

Reply:
[done](https://github.com/openvinotoolkit/training_extensions/pull/2769/commits/1da85964b12868bb793297fab408ea395fe88500)